docker/dsl/build_pipeline.groovy

```diff
@@ -1,10 +1,11 @@
 pipelineJob('build-pipeline-job') { 
 
   // This is the repo that contains the Jenkinsfile you want to use to define your pipeline
-  def repo = 'https://github.com/Synergy232/Special_Topics_Lab_V'
+  def repo = 'https://github.com/Synergy232/Special_Topics_Lab_V/tree/So_i_can_do_a_pull_request_because_i_messed_up_at_the_start'
```

Comment:
You don't want this change... the repo was correctly indicated before (`https://github.com/Synergy232/Special_Topics_Lab_V`).  Your change to line 22 was the correct one (specifying your branch instead of `master`).

Reply:
The builds keep failing on Jenkins and errors almost always branch related. It seems that every answer I find says that what I'm looking for was fixed in a previous update. No matter what changes to the Jenkinsfile, I keep getting nearly the same error. I feel like what I might have to do is to restart the lab and make sure the branches are set properly. Before I do that I wanted to ask is what I have so far even worth 4 points?

----------------most recent error

Started by user unknown or anonymous
Checking out git https://github.com/Synergy232/Special_Topics_Lab_V into /usr/share/jenkins/ref/jobs/build-pipeline-job/workspace@script to read Jenkinsfile
No credentials specified
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/Synergy232/Special_Topics_Lab_V # timeout=10
Fetching upstream changes from https://github.com/Synergy232/Special_Topics_Lab_V
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/Synergy232/Special_Topics_Lab_V +refs/heads/*:refs/remotes/origin/*
 > git rev-parse origin/So_i_can_do_a_pull_request_because_i_messed_up_at_the_start^{commit} # timeout=10
Checking out Revision 37f6faec6617933387e831a1a7e0c3dbd9211976 (origin/So_i_can_do_a_pull_request_because_i_messed_up_at_the_start)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 37f6faec6617933387e831a1a7e0c3dbd9211976
Commit message: "Update Jenkinsfile"
 > git rev-list --no-walk 37f6faec6617933387e831a1a7e0c3dbd9211976 # timeout=10
Running in Durability level: MAX_SURVIVABILITY
[Pipeline] Start of Pipeline
[Pipeline] node
Running on Jenkins in /usr/share/jenkins/ref/jobs/build-pipeline-job/workspace
[Pipeline] {
[Pipeline] stage
[Pipeline] { (checkout sources)
[Pipeline] git
No credentials specified
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/Synergy232/Special_Topics_Lab_V # timeout=10
Fetching upstream changes from https://github.com/Synergy232/Special_Topics_Lab_V
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/Synergy232/Special_Topics_Lab_V +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
 > git rev-parse origin/master^{commit} # timeout=10
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: Couldn't find any revision to build. Verify the repository and branch configuration for this job.
Finished: FAILURE

Reply:
I ended up redoing the lab and have it working now. You should have a new collaborator request.